packages/picasso-rich-text-editor/src/RichTextEditor/story/index.jsx

```diff
@@ -38,3 +38,4 @@ page
     takeScreenshot: false,
   })
   .addExample('RichTextEditor/story/ImageUpload.example.tsx', 'Image upload')
+  .addExample('RichTextEditor/story/Code.example.tsx', 'Code highlight')
```

Comment:
I think we agreed to call it just `code` or `inline code`, while the block version is `code block`. The idea is to also add code blocks to this example?
```suggestion
  .addExample('RichTextEditor/story/Code.example.tsx', 'Inline code')
```

Reply:
done